securedrop_client/logic.py

```diff
@@ -221,10 +221,10 @@ def setup(self):
         self.sync_timer.timeout.connect(self.update_sync)
         self.sync_timer.start(30000)
 
-        # Automagically sync with the API every 5 minutes.
+        # Automagically sync with the API every minute.
         self.sync_update = QTimer()
         self.sync_update.timeout.connect(self.sync_api)
-        self.sync_update.start(1000 * 60 * 5)  # every 5 minutes.
+        self.sync_update.start(1000 * 60)  # every minute.
```

Comment:
this time seems good for now, we can increase sync frequency once all sync_api calls have been removed

Reply:
also: you need to remove the line of code that resumes the queues in on_sync_failure now that sync is outside of the main queue and we don't need to unpause the queue in order to run the MetadataSyncJob. We should never pause the metadata sync queue.

Reply:
one more note: while offline you can stop the thread and restart it on login, which will take care of this issue: https://github.com/freedomofpress/securedrop-client/issues/671 since you can remove the sync_api call on login and instead just start the metadata sync thread

Reply:
I've removed the queue resumption code as requested. The QTimer will kick in (after a minute) and try another MetadataSyncJob.

See my comment in #675